src/tracker/psmove_tracker.c

```diff
@@ -382,7 +390,7 @@ psmove_tracker_remember_color(PSMoveTracker *tracker, struct PSMove_RGBValue rgb
 void
 psmove_tracker_settings_set_default(PSMoveTrackerSettings *settings)
 {
-    *settings = tracker_default_settings; 
+    *settings = create_tracker_default_settings(); 
```

Comment:
If this is the only place where the default settings are used, we could make it so that the function takes a pointer to the object that should be set to defaults. Avoids two object copies. Also the "const" on the return value isn't really useful if returned by value.

```
static void set_tracker_default_settings(PSMoveTrackerSettings *settings)
{
    settings->camera_frame_width = 0;
    settings->camera_frame_height = 0;
    settings->camera_frame_rate = 0;
    settings->camera_auto_gain = PSMove_False;
    settings->camera_gain = 0;
    // ...
}
```

Come to think of it, at this point you could just move the setting code directly to `psmove_tracker_settings_set_default`, getting rid of the extra function:

```
void psmove_tracker_settings_set_default(PSMoveTrackerSettings *settings)
{
    settings->camera_frame_width = 0;
    settings->camera_frame_height = 0;
    settings->camera_frame_rate = 0;
    settings->camera_auto_gain = PSMove_False;
    settings->camera_gain = 0;
    // ...
}
```


Please also take care of indentation, don't mix tabs with spaces, and use 4 spaces indentation.